Tresjs/tres
# feat: pass all provides down to custom renderer

Closes #537 
Closes #732 

This PR adds an opt-out bridge to pass down Vue parent `provides` down to the Tres components via `TresCanvas` with a flag to disable it `enableProvideBridge`

## How to test

- `pnpm run dev`
- Go to [#732: provide/inject](http://localhost:5173/issues/732)
- Open devtools
- Click the button

You should see console logs with prefixes like `tres` from the child tres component with the injects and `vue` for the vue child components, both should show a value different from undefined. 

If you pass the `:enable-provide-bridge="false"` and refresh, the `tres` consoles should be undefined.
